lib/content/message/last_trip/platform_closed.ex

```diff
@@ -11,7 +11,7 @@ defmodule Content.Message.LastTrip.PlatformClosed do
 
   defimpl Content.Message do
     def to_string(%Content.Message.LastTrip.PlatformClosed{}) do
-      "Platform closed"
+      "Service ended"
```

Comment:
**question:** should this be a different struct altogether? should i combine it with `%ServiceEnded{}` (which displays "Service ended for night" or "No <headsign> trains") with some sort of additional configuration?

if we use these as representations of what _kind_ of content to show, sort of the context of the message, i feel like we can keep this as-is?

Reply:
Yeah, I agree we don't need to combine it with the `%ServiceEnded{}` struct. Like you said, the struct names just express the context of the message, which isn't changing, and whatever text they actually render to is just a minor detail imo.